docs/concepts/service-catalog/index.md

```diff
@@ -7,10 +7,10 @@ approvers:
 {% capture overview %}
 {% glossary_definition term_id="service-catalog" length="all" prepend="Service Catalog is" %}  
 
-A *Service Broker*, as defined by the [Open Service Broker API spec](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md), is an endpoint for a set of Managed Services offered and maintained by a third-party, which could be a cloud provider such as AWS, GCP, or Azure.
-Some examples of *Managed Services* are Microsoft Azure Cloud Queue, Amazon Simple Queue Service, and Google Cloud Pub/Sub, but they can be any software offering that can be used by an application.
+A Service Broker, as defined by the [Open Service Broker API spec](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md), is an endpoint for a set of Managed Services offered and maintained by a third-party, which could be a cloud provider such as AWS, GCP, or Azure.
```

Comment:
It provides a way to list, provision, and bind with external managed services from service brokers.

No caps for "managed services". No caps for "service brokers". Unless those things are proper names, but I don't think they are.

Reply:
acknowledged